blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineCreateRequest.java

```diff
@@ -75,11 +75,11 @@ public GithubPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
     @SuppressWarnings("unchecked")
     @SuppressFBWarnings(value = "REC_CATCH_EXCEPTION", justification = "Runtime exception is thrown from the catch block")
     @Override
-    public BluePipeline create(Reachable parent) throws IOException {
+    public BluePipeline create(Reachable parent, @Nonnull BlueOrganization organization) throws IOException {
         Preconditions.checkNotNull(parent, "Parent passed is null");
         Preconditions.checkNotNull(getName(), "Name provided was null");
 
-        User authenticatedUser = checkUserIsAuthenticatedAndHasItemCreatePermission();
+        User authenticatedUser = checkUserIsAuthenticatedAndHasItemCreatePermission(organization);
```

Comment:
I was in doubt if it was best passing the organization to all these methods or have it in a field. But given that we can't initialize it in the constructor, it seems to be safer this way.